public/v4/apps/penpot.yml

```diff
@@ -70,11 +70,8 @@ services:
             - $$cap_appname-assets-data:/opt/data
         environment:
             PENPOT_FLAGS: $$cap_flags
-        caproverExtra:
-            dockerfileLines:
```

Comment:
You've removed `dockerfileLines` so you'll have to add an image or else this service won't be built

Reply:
thanks for pointing that out. fixed.